css-contain-3/Overview.bs

```diff
@@ -303,6 +303,79 @@ Container Queries</h2>
 		will each respond to their own container context.
 	</div>
 
+	For selectors with pseudo elements, query containers can be established by
+	the <a>shadow-including inclusive ancestors</a> of the <a>ultimate
+	originating element</a>.
+
+	It follows that:
```

Comment:
This following section seems like it could be in a note, rather than part of the normative text?

Reply:
Done